packages/expo/src/provider/singleton/createClerkInstance.ts

```diff
@@ -168,12 +168,20 @@ export function createClerkInstance(ClerkClass: typeof Clerk) {
         }
       });
 
+      let nativeApiErrorShown = false;
       // @ts-expect-error - This is an internal API
       __internal_clerk.__unstable__onAfterResponse(async (_: FapiRequestInit, response: FapiResponse<unknown>) => {
         const authHeader = response.headers.get('authorization');
         if (authHeader) {
           await saveToken(KEY, authHeader);
         }
+
+        if (!nativeApiErrorShown && response.payload?.errors?.[0]?.code === 'native_api_disabled') {
+          console.error(
+            'Clerk: The Native API is disabled for this instance.\nGo to Clerk Dashboard > Configure > Native applications to enable it.',
```

Comment:
❓ Should we add a this link `https://dashboard.clerk.com/last-active?path=native-applications` just to make it easier to navigate there?

Reply:
Done!